feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/overview/ConversationList.kt

```diff
@@ -249,12 +247,11 @@ private fun LazyListScope.conversationSectionHeader(
                     Icon(
                         imageVector = if (isExpanded) Icons.Default.ArrowDropDown else Icons.AutoMirrored.Filled.ArrowRight,
                         contentDescription = null,
-                        modifier = Modifier.size(32.dp)
+                        modifier = Modifier.size(32.dp),
+                        tint = MaterialTheme.colorScheme.primaryContainer
```

Comment:
Imo using `primaryContainer` here is quite hard on the eyes, especially in dark mode. Maybe just using the less saturated `primary` would fit better for the toggle icon color, what do you think?
![image](https://github.com/user-attachments/assets/64a3116b-4e48-4d4e-9d15-b6b0f24dd1c8)

Reply:
<img width="293" alt="Screenshot 2024-12-29 at 21 01 30" src="https://github.com/user-attachments/assets/599236a9-e558-46cc-a47a-2c0904f24ff5" />
<img width="293" alt="Screenshot 2024-12-29 at 21 01 16" src="https://github.com/user-attachments/assets/3e77272e-4a2b-4e82-bae9-4e9df4e45956" />

after primary this is the status are we good with that?  I will also update message indicator color

Reply:
My idea was to just use the primary color for the toggle icon, I think the unread indicator could keep using the primaryContainer. But the toggle color looks better now imo :)

Reply:
Only for the arrow, okay I thought we want to manipulate hole section than I will only change that. Thanks for the clarification 😅